doc/configure_customized_buildfarm.rst

```diff
@@ -70,7 +70,7 @@ Change the priority and timeouts of the Jenkins jobs
 ----------------------------------------------------
 
 You can customize the priorities of the generated jobs to fit your needs.
-Depending on the performance of your build slave you might also want yo adjust
+Depending on the performance of your build agent you might also want to adjust
```

Comment:
The spelling error has been fixed on the master branch: e98c37e64ae336b8694650ed2f09152c0ad5a01f Please rebase.